--- kernel/source/include/linux/uts.h	2024-08-27 18:45:52.876098270 +0000
+++ kernel.modified/source/include/linux/uts.h	2024-08-27 18:49:59.588096706 +0000
@@ -6,7 +6,7 @@
  * Defines for what uname() should return 
  */
 #ifndef UTS_SYSNAME
-#define UTS_SYSNAME "Linux"
+#define UTS_SYSNAME "Core"
 #endif
 
 #ifndef UTS_NODENAME
@@ -14,7 +14,7 @@
 #endif
 
 #ifndef UTS_DOMAINNAME
-#define UTS_DOMAINNAME "(none)"	/* set by setdomainname() */
+#define UTS_DOMAINNAME "(none)" /* set by setdomainname() */
 #endif
 
 #endif
